docker-compose-3-persona.yml

```diff
@@ -41,91 +43,132 @@ services:
+        condition: service_healthy
+    ports:
+      - 3002:3000
     environment:
-      - PORT=9000
       - API_HOST=member-a-node
-      - API_PORT=9944
       - DB_HOST=member-a-postgres-identity
       - DB_PORT=5432
       - DB_NAME=sqnc-identity
       - DB_USERNAME=postgres
       - DB_PASSWORD=postgres
       - SELF_ADDRESS=5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
       - IDP_CLIENT_ID=sequence
-      - IDP_PUBLIC_URL_PREFIX=http://localhost:3080/realms/member-a/protocol/openid-connect
-      - IDP_INTERNAL_URL_PREFIX=http://keycloak:8080/realms/member-a/protocol/openid-connect
-    networks: ['member-a']
+      - IDP_PUBLIC_ORIGIN=http://localhost:3080
+      - IDP_INTERNAL_ORIGIN=http://keycloak:8080
+      - IDP_OAUTH2_REALM=member-a
+      - IDP_INTERNAL_REALM=internal
+    networks: [ 'member-a' ]
+
+  member-a-attachment:
+    image: digicatapult/sqnc-attachment-api:v2.1.0
+    container_name: member-a-attachment
+    command: /bin/sh -c "npx knex --env production migrate:latest && npm start"
+    depends_on:
+      member-a-postgres-attachment-api:
+        condition: service_healthy
+    ports:
+      - 3003:3000
+    environment:
+      - DB_HOST=member-a-postgres-attachment
+      - DB_USERNAME=postgres
+      - DB_PASSWORD=postgres
+      - DB_NAME=sqnc-attachment-api
+      - IPFS_HOST=ipfs
+      - IDENTITY_SERVICE_HOST=member-a-identity
+      - IDP_CLIENT_ID=sequence
+      - IDP_PUBLIC_ORIGIN=http://localhost:3080
+      - IDP_INTERNAL_ORIGIN=http://keycloak:8080
+      - IDP_OAUTH2_REALM=member-a
+      - IDP_INTERNAL_REALM=internal
+    networks: [ 'member-a', 'ipfs' ]
 
   member-a-node:
-    image: digicatapult/sqnc-node:latest
+    image: digicatapult/sqnc-node:v12.3.0
     container_name: member-a-node
-    command: --base-path /data
-      --alice
-      --unsafe-rpc-external
-      --rpc-cors all
-      --node-key 0000000000000000000000000000000000000000000000000000000000000001
+    command: --base-path /data --alice --unsafe-rpc-external --rpc-cors all --node-key 0000000000000000000000000000000000000000000000000000000000000001
     ports:
-      - 30333:30333
       - 9944:9944
-      - 9933:9933
     restart: on-failure
-    networks: ['member-a', 'chain']
+    volumes:
+      - member-a-node-storage:/data
+    networks: [ 'member-a', 'chain' ]
 
   member-a-matchmaker-api:
-    container_name: member-a-matchmaker-api
+    container_name: member-a-matchmaker
     build:
       context: './'
       dockerfile: 'Dockerfile'
-    command: /bin/sh -c "
-      sleep 10 &&
-      npx knex migrate:latest --knexfile build/lib/db/knexfile &&
-      npx @digicatapult/sqnc-process-management@latest create -h member-a-node -p 9944 -u //Alice -f processFlows.json &&
-      npm start"
+    command: /bin/sh -c "npx knex migrate:latest --knexfile build/lib/db/knexfile && npx @digicatapult/sqnc-process-management@latest create -h member-a-node -p 9944 -u //Alice -f processFlows.json && npm start"
```

Comment:
as you're pinning versions elsewhere, do you also want to pin `sqnc-process-management` ?

Reply:
will update